config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-presubmit-dra-capz.yaml

```diff
@@ -181,7 +181,7 @@ presubmits:
         - name: WINDOWS_WORKER_MACHINE_COUNT
           value: "0" # Don't create windows workers
         - name: WORKER_MACHINE_COUNT
-          value: "100"
+          value: "1000"
```

Comment:
again, not able to comment but can you do 
```
        - name: CL2_NODES_PER_NAMESPACE
          value: "100"
```

so we dont 10x the number of namespaces created by the test